openapi_python_client/templates/model.pyi

```diff
@@ -58,18 +63,13 @@ class {{ model.reference.class_name }}:
         field_dict.update(self.additional_properties)
         {% endif %}
         {% endif %}
-        field_dict.update({
```

Comment:
The author of the original template was trying to save a little space I guess, by using `update` instead of setting the properties one by one, but since a required property could be read-only we can't assume we will always want to set them all.